### ISSUE TITLE
Error when updating to 2.11.5

### ISSUE THREAD

**Issue by edalzell** [statamic/v2-hub#2328]
**Describe the bug**
Get `Could not authenticate against github.com ` when updater is updating dependencies

**To Reproduce**
1. Have at least one addon with dependencies
2. Be on Statamic < 2.11.5
3. `statamic update`

**Expected behavior**
Site is updated

**Screenshots**
https://cl.ly/b2f22559a3d9

**Environment details (please complete the following information):**
 - Statamic Version 2.11.4
 - Fresh Install or Upgrade: upgrade
 - OS: macOS Mojave
 - PHP Version: 7.2
 - Addons installed: hahaha...lots

**Comment by subpixelch** [comment]
same here:
```
➜  statamic_standard git:(master) ✗ php please update:addons
Adding packages: subpixel/assetspro, marbles/cookie-consent, rap2hpoutre/laravel-log-viewer, marbles/sitemap, marbles/Spaceless
Please wait while dependencies are updated via Composer. This may take a while.
The authenticity of host 'github.com (140.82.118.3)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? yes
The command "php composer.phar update subpixel/assetspro marbles/cookie-consent rap2hpoutre/laravel-log-viewer marbles/sitemap marbles/Spaceless --prefer-dist --no-dev --optimize-autoloader" failed.
Exit Code: 1(General error)

Output:
================

Error Output:
================
Loading composer repositories with package information
Failed to clone the git@github.com:statamic/Stringy.git repository, try running in interactive mode so that you can enter your GitHub credentials

  [RuntimeException]
  Failed to execute git clone --mirror 'git@github.com:statamic/Stringy.git' '/Users/patrick/Desktop/websites/statamic_standard/statamic/../local/composer/cache/vcs/git-github.com-statamic-Stringy.git/'

Dependencies updated!
```

**Comment by jackmcdade** [comment]
Fixed with [2.11.6](https://statamic.com/changelog#2.11.6) - my apologies on that one. We needed to publish an official Packagist fork of Stringy, not just use a `vcs` Composer override. One more update and you should be good to go!
